bin/node/runtime/src/lib.rs

```diff
@@ -639,12 +639,12 @@ impl pallet_treasury::Trait for Runtime {
 	type ApproveOrigin = EnsureOneOf<
 		AccountId,
 		EnsureRoot<AccountId>,
-		pallet_collective::EnsureMembers<_4, AccountId, CouncilCollective>
+		pallet_collective::EnsureProportionAtLeast<_3, _5, AccountId, CouncilCollective>
 	>;
 	type RejectOrigin = EnsureOneOf<
 		AccountId,
 		EnsureRoot<AccountId>,
-		pallet_collective::EnsureMembers<_2, AccountId, CouncilCollective>
+		pallet_collective::EnsureProportionMoreThan<_1, _2, AccountId, CouncilCollective>
```

Comment:
just a note asking for 2 members seems very different from asking for half members

Reply:
Not if the number of members is 4 :)

I just picked what was in Polkadot, which makes sense to me :)

Reply:
true :-) indeed propostion make more sense